x-pack/plugins/enterprise_search/public/plugin.ts

```diff
@@ -17,7 +16,7 @@ import {
   FeatureCatalogueCategory,
   HomePublicPluginSetup,
 } from '../../../../src/plugins/home/public';
-import { LicensingPluginSetup } from '../../licensing/public';
+import { LicensingPluginStart } from '../../licensing/public';
```

Comment:
Extra context for this change: https://github.com/elastic/kibana/blob/5f781dcfecc9bcd96d9fa5ecfe5cd3e1cdedf094/x-pack/plugins/licensing/public/types.ts#L12-L17

I noticed it because VSCode was showing the old license$ with a strikethrough through it because of the @deprecated. Pretty cool!